README.md

```diff
@@ -7,26 +7,13 @@ Ruby LSP Rails is a [Ruby LSP](https://github.com/Shopify/ruby-lsp) addon for ex
 
 ## Installation
 
-To install, add the following line to your application's Gemfile:
-
-```ruby
-# Gemfile
-group :development do
-  gem "ruby-lsp-rails"
-end
-```
+As of v0.3.0, [Ruby LSP](https://github.com/Shopify/ruby-lsp) will automatically activate the Ruby LSP Rails addon when a Rails app is detected. There is no need to add the gem to your bundle.
 
 ## Usage
 
 ### Hover to reveal ActiveRecord schema
 
-1. Start your Rails server
-1. Hover over an ActiveRecord model to see its details
-
-### Documentation
```

Comment:
I moved this out of the Usage section since its about the overall documentation for Ruby LSP Rails rather than the Rails Documentation feature.